addons/website_sale_digital/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
useless, since py3 it is utf-8 by default and all editors/ides use utf-8 by default